spec/requests/api/v1/job_applications/job_application_create_spec.rb

```diff
@@ -92,6 +106,23 @@
         expect(json[:message]).to eq("Company must exist and Position title can't be blank")
         expect(json[:status]).to eq(400)
       end
+      
+      it "returns a error message if a user tries to create multiple job applications with the same URL" do
+        post "/api/v1/users/#{@user.id}/job_applications", 
+        params: { job_application: job_application_params }
+        expect(response).to be_successful
+        expect(response.status).to eq(200)
+
+        post "/api/v1/users/#{@user.id}/job_applications", 
+        params: { job_application: job_application_params }
+        expect(response).to_not be_successful
+        expect(response.status).to eq(400)
+
+        json = JSON.parse(response.body, symbolize_names: true)
+
+        expect(json[:message]).to eq("Application url You already have an application with this URL")
```

Comment:
Informative message to the user, guides them to a happy path gracefully. At the risk of being nitpicky, would we want to restructure the error message to read closer to a sentence or a structured message? I dont know if mentioning the failing validation is useful to the end user.
ex) "Error: Application url. You already have an application with this URL" or just "You already have an application with this URL""

Reply:
This is a good point, rails has a default configuration for formatting uniqueness validations, which resulted in the oddly phrased message. I will update so it reads more clearly. Good callout :)

Reply:
Just rephrased the error message, and fixed tests to align. If this change is acceptable please mark this conversation as resolved.